test_data/rules/unsafe_unicode.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `dart test -N unsafe_unicode`
+
+var u202e = '‮'; // LINT [-6]
```

Comment:
Cool to see how this (mis)renders.

Here's how it looks in the IDE:

![image](https://user-images.githubusercontent.com/67586/139920599-e4425a5a-433c-4f52-a9ac-8eb55216cdc8.png)


A bit of a case in point eh?